curriculum/challenges/english/08-coding-interview-prep/rosetta-code/state-name-puzzle.md

```diff
@@ -6,19 +6,16 @@ challengeType: 5
 
 ## Description
 <section id='description'>
-<b>Background</b>
-This task is inspired by <a href="http://drdobbs.com/windows/198701685">Mark Nelson's DDJ Column "Wordplay"</a> and one of the weekly puzzle challenges from Will Shortz on NPR Weekend Edition [http://www.npr.org/templates/story/story.php?storyId=9264290] and originally attributed to David Edelheit.
+This task is inspired by <a href="http://drdobbs.com/windows/198701685" target="_blank">Mark Nelson's DDJ Column "Wordplay"</a> and one of the weekly puzzle challenges from Will Shortz on NPR Weekend Edition <a href="http://www.npr.org/templates/story/story.php?storyId=9264290"target="_blank">[1]</a> and originally attributed to David Edelheit.
```

Comment:
The drdobbs.com link does not have an ssl version, so it should be removed.  The npr.org link should be changed to `https`.  Also, you should probably put a space before  `target="blank"` in the npr.org anchor tag.